src/main/java/org/opensearch/neuralsearch/processor/AgenticQueryTranslatorProcessor.java

```diff
@@ -74,8 +75,18 @@ public void processRequestAsync(
             return;
         }
 
+        AgenticSearchQueryBuilder agenticQuery = (AgenticSearchQueryBuilder) query;
+
+        // Agentic search request initiated
+        log.info("Agentic search request initiated - Agent ID: [{}], Query: [{}]", agentId, agenticQuery.getQueryText());
```

Comment:
```suggestion
        log.info("Agentic search request initiated - Agent ID: [{}], Query: [{}]", agentId, agenticQuery.getQueryText());
```

Can we use String.format here?

Reply:
Parameterized logs are suited better than String.format. Any reason for String.format?

Reply:
Its fine with what you have now. Please check following comments whether we really need to keep these log messages seperately